Feature/command migration locale migration

RainbowDashLabs opened the pull request:
All changes regarding the locale code renaming. Additionally English_UK is now supported. Portuguese is no longer supported but the Brazilian,

Taucher2003 commented on src/main/resources/locale_es_ES.properties:
```diff
@@ -258,45 +257,44 @@ command.scan.start.message.queueFull=Se están ejecutando demasiados escaneos ac
 command.thankwords.loaddefault.message.available=Idiomas disponibles:
 command.thankwords.remove.description=Eliminar un patrón
-command.thankwords.remove.pattern.description=El patrón regex a eliminar.
+command.thankwords.remove.message.patternnotnound=El patrón no existe.
```
Looks like there is a typo in the key

Taucher2003 commented on src/main/resources/locale_pt_BR.properties:
```diff
@@ -0,0 +1,361 @@
+command.abuseprotection.donorcontext.description=Obtenha ou defina se o doador precisa ou não de mensagens recentes.
+command.abuseprotection.donorcontext.message.false=O doador não precisa de mensagens recentes no canal.
+command.abuseprotection.donorcontext.message.true=O doador deve ter mensagens recentes no canal.
+command.abuseprotection.donorcontext.state.description=Defina como "verdadeiro" para ativar as verificações de contexto do doador.
```
I think the quoted text should be `true`?
```suggestion
command.abuseprotection.donorcontext.state.description=Defina como "true" para ativar as verificações de contexto do doador.
```

Taucher2003 commented on src/main/resources/locale_pt_BR.properties:
```diff
@@ -0,0 +1,361 @@
+command.roles.stackroles.description=Permitir o empilhamento de papéis
+command.roles.stackroles.message.notStacked=O usuário só terá o papel mais próximo de sua reputação.
+command.roles.stackroles.message.stacked=Os papéis serão empilhados. Os usuários terão todos os papéis com menos do que a sua reputação.
+command.roles.stackroles.stack.description=Definido como "verdadeiro" para empilhar.
```
```suggestion
command.roles.stackroles.stack.description=Definido como "true" para empilhar.
```

Taucher2003 reviewed (commented):
en_US and en_GB are completely identical, what is the value of having both?

RainbowDashLabs commented:
@Taucher2003 I removed the gb file for now, but maybe some day we will have actual differences. But if this ever happens we will just readd it again.

I also noticed that we need to rename a couple of locale codes as well and added them to the migration script
